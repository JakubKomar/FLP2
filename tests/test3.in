A B
C D
C E
A C
A D
A E
A F
C F
D E
D F
B C
B D
B E
B F