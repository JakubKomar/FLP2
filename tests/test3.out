./flp22-log
A-B A-C B-E C-F D-E D-F 
A-B A-C B-F C-E D-E D-F 
A-B A-D B-E C-E C-F D-F 
A-B A-D B-F C-E C-F D-E 
A-B A-E B-C C-F D-E D-F 
A-B A-E B-D C-E C-F D-F 
A-B A-E B-F C-D C-E D-F 
A-B A-E B-F C-D C-F D-E 
A-B A-F B-C C-E D-E D-F 
A-B A-F B-D C-E C-F D-E 
A-B A-F B-E C-D C-E D-F 
A-B A-F B-E C-D C-F D-E 
A-C A-D B-E B-F C-E D-F 
A-C A-D B-E B-F C-F D-E 
A-C A-E B-C B-F D-E D-F 
A-C A-E B-D B-E C-F D-F 
A-C A-E B-D B-F C-F D-E 
A-C A-E B-E B-F C-D D-F 
A-C A-F B-C B-E D-E D-F 
A-C A-F B-D B-E C-E D-F 
A-C A-F B-D B-F C-E D-E 
A-C A-F B-E B-F C-D D-E 
A-D A-E B-C B-E C-F D-F 
A-D A-E B-C B-F C-E D-F 
A-D A-E B-D B-F C-E C-F 
A-D A-E B-E B-F C-D C-F 
A-D A-F B-C B-E C-F D-E 
A-D A-F B-C B-F C-E D-E 
A-D A-F B-D B-E C-E C-F 
A-D A-F B-E B-F C-D C-E 
A-E A-F B-C B-D C-E D-F 
A-E A-F B-C B-D C-F D-E 
A-E A-F B-C B-E C-D D-F 
A-E A-F B-C B-F C-D D-E 
A-E A-F B-D B-E C-D C-F 
A-E A-F B-D B-F C-D C-E 
